src/Microsoft.AspNetCore.Mvc.Core/BindPropertyAttribute.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.AspNetCore.Mvc
 {
-    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Property, AllowMultiple = false, Inherited = true)]
+    [AttributeUsage(AttributeTargets.Property, AllowMultiple = false, Inherited = true)]
```

Comment:
API check won't catch changes in (especially reductions of) the `[AttributeUsage]`. Suggest filing an issue tracking that.

Reply:
Add a comment about _not_ supporting this attribute on a class because this is the second time it's been removed. See 1886d53d89

Reply:
Filed an issue for the ApiCheck. The doc comment has marginal value, leaving it alone.

Reply:
thanx